git_machete/client.py

```diff
@@ -2021,7 +2021,9 @@ def create_github_pr(
         debug(f'create_github_pr({head})', f'organization is {org}, repository is {repo}')
         debug(f'create_github_pr({head})', 'current GitHub user is ' + (current_user or '<none>'))
 
-        title: str = self.__git.get_commit_information("subject").strip()
+        fork_point = self.__git.get_merge_base(self.__git.get_commit_sha_by_revision(head), self.__git.get_commit_sha_by_revision(base))
```

Comment:
Fork point should rather be computed using `git_machete.client.MacheteClient.fork_point`!

See https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02 (esp. the appendix)

The concept is an improvement over what `git rebase --fork-point` does &mdash; basically, to determine **after which commit the unique history** of the given branch started. Or in other words, what's the latest ancestor commit of the given branch that has been committed while a different branch has been checked out.

This will indeed be sometimes equal to `merge_base(branch, parent(branch))` (e.g. in all green-edge cases)... but in many cases (esp. red edges) it isn't! Feel free to ask further questions &mdash; this concept is pretty non-trivial since git does NOT assign commits to branches - branches do NOT contain commits, branches only **point to** commits. So finding the set of commits "uniquely belonging" to a branch is working around git data model... although still quite useful ;)